context.go

```diff
@@ -3208,7 +3208,7 @@ func (c *Context) writeLocalBuildActions(nw *ninjaWriter,
 	return nil
 }
 
-func beforeInModuleList(a, b *moduleInfo, list []*moduleInfo) bool {
+func aPrecedesBInModuleList(a, b *moduleInfo, list []*moduleInfo) bool {
```

Comment:
I agree the original is hard to parse, but I don't think this helps because any callers don't have an "a" and a "b", those are local to the function.  Maybe just add godoc that describes that this checks if the first argument is before the second argument in the list?